learning-development/accessibility.qmd

```diff
@@ -51,22 +51,61 @@ For more advice about colour in charts and visualisations specifically, includin
 
 ---
 
+## Assistive technology
+
+No automated tools fully cover accessibility, manual testing is always required, so if you want to do manual testing have a look at testing out the assistive software commonly used by users yourself.
```

Comment:
I would probably change the first sentence of this - theoretically it's very difficult to make a site accessible for _everyone_ (there's lots of discussion about this) so I'd probably say something like "Manual testing is almost always required to make sure that your content is compliant with WCAG 2.2"

Reply:
Much better, thanks